src/easylink/implementation_metadata.yaml

```diff
@@ -233,132 +233,132 @@ step_1a_and_step_1b_combined_python_pandas:
   - step_1a
   - step_1b
   image_name: python_pandas.sif
-  zenodo_record_id: 15611084
-  md5_checksum: 7cc7cb37195c635684903b6777cf1cdf
+  zenodo_record_id: 15733426
+  md5_checksum: 9177b8e168fcc9cae91bf61265f2185c
   script_cmd: python /dummy_step.py
   outputs:
     step_1_main_output: result.parquet
 dummy_step_1_for_output_dir_example:
   steps:
   - step_1_for_output_dir_example
-  image_name: main/dummy_step_1_for_output_dir_example.sif
+  image_name: sbachmei/dummy_step_1_for_output_dir_example.sif
   script_cmd: python /dummy_step_1_for_output_dir_example.py
   outputs:
     step_1_main_output_directory: output_dir/
 dummy_step_1_for_output_dir_example_default:
   steps:
   - step_1_for_output_dir_example
-  image_name: main/dummy_step_1_for_output_dir_example.sif
+  image_name: sbachmei/dummy_step_1_for_output_dir_example.sif
   script_cmd: python /dummy_step_1_for_output_dir_example.py
 dummy_step_2_for_output_dir_example:
   steps:
   - step_2_for_output_dir_example
-  image_name: main/dummy_step_2_for_output_dir_example.sif
+  image_name: sbachmei/dummy_step_2_for_output_dir_example.sif
```

Comment:
Is your name suppose to be here?

Reply:
yes, it's our weird development workflow. once this gets fully approved I'll copy the containers to the main/ dir and change all of these appropriately back